Makefile

```diff
@@ -0,0 +1,35 @@
+.DEFAULT_GOAL := test-all
+
+.PHONY: test-all
+test-all: lint test
+
```

Comment:
```suggestion
```

This `make test-all` feels confusing to me (because it doesn't run enterprise test despite saying `all`) and is not significantly shorter than `make lint test`.